index.ts

```diff
@@ -61,7 +61,9 @@ async function fetchGif(
       return null;
     }
   } catch (e) {
-    core.debug(e.message);
+    if (e instanceof Error) {
+      core.debug(e.message);
```

Comment:
Is there any high level output, like warn or error?

Reply:
It's been a while since I've looked at the docs for this, but I feel that for a fun action like this, it'd be better to noop fail than add logs, unless explicitly looking for debug logs.